dao/mongo.go

```diff
@@ -273,3 +274,37 @@ func (m *MongoService) GetPaymentsWithRefundStatus() ([]models.PaymentResourceDB
 
 	return payments, nil
 }
+
+// GetPaymentsWithPaidStatus retrieves a list of all payments in the DB with a status of PAID
+func (m *MongoService) GetPaymentsWithPaidStatus() ([]models.PaymentResourceDB, error) {
```

Comment:
Rename to something like `GetPaymentsWithRefundPendingStatus`

Reply:
Fixed.Thanks